test/fixtures/test-runner/no-isolation/a.test.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+const test = require('node:test');
+
+test('a', (t, cb) => { globalThis.aCb = cb; });
```

Comment:
nit: can we have EOL for those fixtures files
```suggestion
test('a', (t, cb) => { globalThis.aCb = cb; });

```